File: dataset/human/cpp/file_1386.cpp
#include <iostream>
using namespace std;
 
int main() {
    int r, g, b, n;
    cin >> r >> g >> b >> n;
    int ans = 0;
 
    for (int i = 0; i < n + 1; i++) {
        if (r*i > n) 
            continue;
 
        for (int j = 0; j < n + 1; j++) {
            if (r*i + g*j > n) 
                continue;
            if ((n - r*i - g*j) % b == 0) 
                ans++;
        }
    }
 
    cout << ans << endl;
    return 0;
}

File: dataset/human/python/file_1386.py
import sys
stdin = sys.stdin

ni = lambda: int(ns())
na = lambda: list(map(int, stdin.readline().split()))
nn = lambda: list(stdin.readline().split())
ns = lambda: stdin.readline().rstrip()

n = ni()
a = na()

import numpy as np
a = list(np.cumsum(a))

ans = 100000000000000

for i in range(n-1):
    ans = min(ans, abs(a[-1]-a[i]*2))

print(ans)
